parts/linux/cloud-init/artifacts/ci-syslog-watcher.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o pipefail
+
+[ ! -f "/var/run/mdsd/update.status" ] && exit 0
+status=$(cat /var/run/mdsd/update.status)
+
+if [[ "$status" == "add" ]]; then
+        echo "Status changed to $status."
+        cat >/etc/rsyslog.d/60-rsyslog-forward-mdsd.conf << 'EOL'
+$ModLoad omuxsock 
+$OMUxSockSocket /var/run/mdsd/default_syslog.socket 
```

Comment:
Cant we configure the MDSD to create and listen on specific socket instead of using default socket which could avoid any potentially conflict with MDSD running on the host?

Reply:
yes we will use something like /var/run/mdsd-ci/default_syslog.socket